serverlessworkflow/modules/ROOT/pages/cloud/deploying-on-minikube.adoc

```diff
@@ -1 +1,394 @@
-//= Deploying on Minikube
\ No newline at end of file
+= Deploying on Minikube
+=====================================================
+06-22-2022
+:compat-mode!:
+// Metadata:
+:description: Deploying Serverless Application on Minikube
+:keywords: kogito, workflow, quarkus, serverless, kn, kubectl, minikube
+:table-caption: Data Set
+// links
+:minikube_url: https://minikube.sigs.k8s.io
+:kn_cli_url: https://knative.dev/docs/install/client/
+:kn_cli_quickstart_plugin_url: https://knative.dev/docs/install/quickstart-install/#install-the-knative-cli
+:knative_on_minikube_step_by_step_url: https://redhat-developer-demos.github.io/knative-tutorial/knative-tutorial/setup/minikube.html
+:knative_issue_url: https://github.com/knative/serving/issues/6101
+
+
+This document describes how to deploy a Serverless Application using a local Kubernetes cluster link:{minikube_url}}[Minikube] as well the steps required to get the Knative plataform running. Please make sure to read the Get Started Minikube guide and assure that your machine meets the System Requirement.
```

Comment:
Where's the guide link? In the latest link in this paragraph?

Reply:
which link?